projects/opendr_ws/src/perception/scripts/facial_emotion_estimation.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
```

Comment:
The location for the ROS scripts has been changed, this file should be under: opendr/projects/opendr_ws/opendr_perception

Reply:
I moved it and also applied several fixes on the node, so i mark this as resolved @negarhdr